e2e/terraform/ecs-task.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "command": [
+      "/bin/sh -c \"echo '<html> <head> <title>Amazon ECS Sample App</title> <style>body {margin-top: 40px; background-color: #333;} </style> </head><body> <div style=color:white;text-align:center> <h1>Amazon ECS Sample App</h1> <h2>Congratulations!</h2> <p>Your application is now running on a container in Amazon ECS.</p> </div></body></html>' >  /usr/local/apache2/htdocs/index.html && httpd-foreground\""
```

Comment:
Not only that, but this task definition is deployed via terraform instead of via `nomad run`. Nomad only _invokes_ it. James and I always intended to try to make it more like the Docker driver (specify all runtime parameters in the *job* file, not in an external task definition), but we didn't get there with this initial spike.